lnwallet/channel.go

```diff
@@ -1671,14 +1657,15 @@ func (lc *LightningChannel) restoreCommitState(
 		// corresponding state for the local commitment chain.
 		pendingCommitPoint := lc.channelState.RemoteNextRevocation
 		pendingRemoteCommit, err = lc.diskCommitToMemCommit(
-			false, true, &pendingRemoteCommitDiff.Commitment,
+			false, &pendingRemoteCommitDiff.Commitment,
 			nil, pendingCommitPoint,
 		)
 		if err != nil {
 			return err
 		}
 		lc.remoteCommitChain.addCommitment(pendingRemoteCommit)
 
+		// TODO: add htlcs
```

Comment:
They're already logged in the contract court FWIW.

Reply:
woops, stray TODO. Removed.